### DISCUSSION TITLE
Inconsistency in applying Permissions for Editing Pages without access to Move/Rename

### DISCUSSION THREAD

**Discussion by sean-freeman** [requarks/wiki#3319] [Error / Bug Report]
**Describe the bug**

Group Permissions `write:pages` is "create / edit pages" and `manage:pages` is "move existing pages".

However, under Group Page Rules `write:pages` is "Create Pages" and `manage:pages` is "Edit + Move Pages".

*These roles should always be separate.*

There is a use case for allowing a User to edit pages but not be able to rename or move directories.

In the current circumstance the Group Page Rules, to allow the editing of pages also allows a User to rename the pages.

**Comment by maldiavolo** [comment]
Also having this issue.  The way permissions are coded only makes sense in the narrow way it was thought out.  Fine grain permissions is what everyone needs and expects in today's environments.  Read, create, edit, move, and delete must be separate assignable functions.
